src/js/facility-locator/reducers/facilities.js

```diff
@@ -1,5 +1,3 @@
-import { FETCH_VA_FACILITY, FETCH_VA_FACILITIES, SEARCH_FAILED } from '../actions';
```

Comment:
In theory, they'd be useful to share between the usage here and the tests, but I don't anticipate much scaling for FL, so I'm not opposed to using hard-coded strings in two places.